ee/ucp/admin/configure/run-only-the-images-you-trust.md

```diff
@@ -6,55 +6,49 @@ keywords: ucp, dtr, security, trust
+that is not trusted. If the image is not trusted, UCP will not continue with the deployment.
 
 ![Enforce image signing](../../images/run-only-the-images-you-trust-1.svg)
 
-By signing and verifying the Docker images, you ensure that the images being
-used in your cluster are the ones you trust and haven’t been altered either in
-the image registry or on their way from the image registry to your UCP cluster.
+By signing and verifying Docker images, you ensure that:
 
-## Example workflow
+* The images used in your cluster are ones that you trust.
+* The images have not been altered either in the image registry or on their way from the image registry to your UCP cluster.
 
-Here's an example of a typical workflow:
+## Example workflow
 
-1. A developer makes changes to a service and pushes their changes to a version
-   control system.
-2. A CI system creates a build, runs tests, and pushes an image to DTR with the
-   new changes.
-3. The quality engineering team pulls the image and runs more tests. If
-   everything looks good they sign and push the image.
-4. The IT operations team deploys a service. If the image used for the service
-   was signed by the QA team, UCP deploys it. Otherwise UCP refuses to deploy.
+1. A developer makes changes to a service and pushes the changes to a version control system.
+2. A continuous integration (CI) system creates a build, runs tests, and pushes an image to Docker Trusted Registry (DTR) with the new changes.
+3. The quality engineering team pulls the image and runs more tests. If the tests are successful, the team signs and then pushes the image.
+4. The IT operations team deploys a service. If the image used for the service was signed by the QA team, UCP deploys it. Otherwise, UCP refuses to deploy the image.
 
 ## Configure UCP
 
-To configure UCP to only allow running services that use Docker images you
-trust, go to the UCP web UI, navigate to the **Admin Settings** page, and in
-the left pane, click **Docker Content Trust**.
+To configure UCP to only allow running services that use Docker trusted images:
+
+1. From the UCP web UI, navigate to the **Admin Settings** page.
+2. In the left navigation pane, click **Docker Content Trust**.
+3. Select the **Run only signed images** option.
 
-Select the **Run Only Signed Images** option to only allow deploying
-applications if they use images you trust.
+    ![UCP settings](../../images/run-only-the-images-you-trust-2.png){: .with-border}
 
-![UCP settings](../../images/run-only-the-images-you-trust-2.png){: .with-border}
+    With this setting, UCP allows deploying any image as long as the image has
+    been signed. It doesn't matter who signed the image.
 
-With this setting, UCP allows deploying any image as long as the image has
-been signed. It doesn't matter who signed the image.
+    To enforce that the image needs to be signed by specific teams, click **Add Team**
+    and select those teams from the list.
```

Comment:
Suggest editing to read: "To enforce the requirement that the image be signed by specific teams, click **Add Team** and select the pertinent teams from the list."